### ISSUE TITLE
Predictions for unseen data not supported (word-level QE)

### ISSUE THREAD

**Issue by fredblain** [sheffieldnlp/deepQuest#7]
#### Summary
To produce word-level predictions on test data, one currently has to create artificial dummy labels to use along with the data during training. This is inconvenient as it prevents from using a pretrained model to produce labels for unseen data.

#### Expected behavior
One should be able to reuse a pretrained word-level QE model to predict word-level labels for unseen data, using the 'sampling' mode.

#### Related issue(s): #3

**Comment by fredblain** [comment]
Tested with:
- word-level (BiRNN) QE model (aka 'EncWord');
- backend: Tensorflow.

To produce predictions for unseen data: 
- use it as test data;
- add it to 'TEXT_FILES'  -- (e.g.  TEXT_FILES = {'train': 'train.', 'val': 'dev.', **'test': 'test.'**})
- add it to 'EVAL_ON_SETS' -- (e.g. EVAL_ON_SETS = ['test']);
- set 'NO_REF' to True;
- set 'MODE' to 'sampling'.

The path to the file which contains the predictions will be given in the log.
